docs/02.API-REFERENCE.md

```diff
@@ -3787,11 +3787,13 @@ is no longer needed.
 jerry_value_t
 jerry_exec_snapshot (const uint32_t *snapshot_p,
                      size_t snapshot_size,
+                     size_t func_index,
```

Comment:
I have been thinking about it myself. This is a rarely used function, and only an extra 0 needs to be passed, so introducing another function provides little gain. Furthermore if we want to make this function obsolete, and remove it eventually, we have the same problem again, moving from a longer function name to the shorter function name.

However if people insist on it I can add another function.